src/schemas/filtersContainer.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "target": {
+      "$ref": "#target"
+    },
+    "filters": {
+      "type": "array",
+      "items": {
+        "$ref": "#filter"
+      }
+    }
+  },
+  "required": [
+    "filters"
```

Comment:
Yea, I assumed so.  Our old methods had optional target and if it wasn't specified it defaulted to /report/filters instead of /report/page/xyz/filters but now we don't really have need for /report/pages/xyz/filters so it's less important. We could define a target for the report and make the target required if you want.